docs/keystore.md

```diff
@@ -1 +1,117 @@
 # Keystore management
+
+## Key types
+
+There are four types of key representations that are used:
+
+- `cyber` a.k.a. address. Is used for linking content, sending liquid tokens, delegating, redelegating, and undelegating tokens to validators. Also, withdrawing rewards and voting.
+  - Derived from account keys generated by `cyberdcli keys add`
+  - e.g. `cyber15h6vd5f0wqps26zjlwrc6chah08ryu4hzzdwhc`
+
+- `cybervaloper` validator address. Is possible to delegate, redelegate and undelegate liquid tokens to/from this address abd invoke staking commands. Used to associate a validator to its operator
```

Comment:
Please, split keys into 2 groups:
- agent
- validator

For every group clear way of backup and restore must be provided. Now this 3 things are mixed in a text

Reply:
@xhipster add some fixes
take a look please